rpmspec/ublue-os-polkit-rules.spec

```diff
@@ -0,0 +1,36 @@
+Name:           ublue-os-polkit-rules
+Packager:       ublue-os
+Vendor:         ublue-os
+Version:        0.1
+Release:        1%{?dist}
+Summary:        Additional polkit rules
+
+License:        MIT
+URL:            https://github.com/ublue-os/config
+
+BuildArch:      noarch
+Supplements:    systemd-udev
+
+Source0:        ublue-os-polkit-rules.tar.gz
+
+%global sub_name %{lua:t=string.gsub(rpm.expand("%{NAME}"), "^ublue%-os%-", ""); print(t)}
+
+%description
+Adds various polkit rules for improving device support
+
+%prep
+%setup -q -c -T
+
+%install
+tar xf %{SOURCE0} -C %{buildroot}%{_datadir}/%{VENDOR} --strip-components=1
```

Comment:
I think this may be causing the CI problems. From the logs

```
tar xf /tmp/ublue-os/rpmbuild/SOURCES/ublue-os-polkit-rules.tar.gz -C /tmp/ublue-os/rpmbuild/BUILDROOT/ublue-os-polkit-rules-0.1-1.fc39.x86_64/usr/share/ublue-os --strip-components=1
tar: /tmp/ublue-os/rpmbuild/SOURCES/ublue-os-polkit-rules.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
```

Reply:
It's been a long time since I packaged anything in RPM. @storopoli Do you know why it may not be creating the tar?